planning/behavior_velocity_planner/src/scene_module/blind_spot/scene.cpp

```diff
@@ -233,8 +236,15 @@ bool BlindSpotModule::generateStopLine(
       RCLCPP_DEBUG(logger_, "No intersection enter point found.");
       return false;
     }
-    planning_utils::calcClosestIndex(
-      path_ip, intersection_enter_point_opt.get(), stop_idx_ip, 10.0);
+
+    geometry_msgs::msg::Pose intersection_enter_pose;
+    intersection_enter_pose.position = intersection_enter_point_opt.get();
+    const auto stop_idx_ip_opt =
+      motion_utils::findNearestIndex(path_ip.points, intersection_enter_pose, 10.0, M_PI_4);
+    if (stop_idx_ip_opt) {
+      stop_idx_ip = stop_idx_ip_opt.get();
+    }
+
     stop_idx_ip = std::max(stop_idx_ip - base2front_idx_dist, 0);
```

Comment:
@takayuki5168 
I suggest this change and remove L 248 inorder not to use uninitialized value of stop_idx_ip
```suggestion
    if (stop_idx_ip_opt) {
      stop_idx_ip = stop_idx_ip_opt.get();
      stop_idx_ip = std::max(stop_idx_ip - base2front_idx_dist, 0);
    }

```

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/1413/commits/5ebc2475a7bcbd18d61e076ea4776e54cc4a93db
DONE

Reply:
:+1: